jenkins/ci.opensuse.org/openstack-cleanvm.yaml

```diff
@@ -40,7 +41,7 @@
         || ( openstack_project=="Ocata" && (image=="SLE_12_SP3" || image=="openSUSE-Leap-42.3"))
         || ( openstack_project=="Pike" && (image=="SLE_12_SP3" || image=="openSUSE-Leap-42.3"))
         || ( openstack_project=="Queens" && (image=="SLE_12_SP3" || image=="openSUSE-Leap-42.3" || image=="openSUSE-Leap-15.0"))
-        || ( openstack_project=="Master" && (image=="SLE_12_SP3" || image=="openSUSE-Leap-42.3" || image=="openSUSE-Leap-15.0"))
+        || ( openstack_project=="Master" && (image=="SLE_12_SP3" || image=="SLE_12_SP4" || image=="openSUSE-Leap-42.3" || image=="openSUSE-Leap-15.0"))
```

Comment:
image="SLE_12_SP4" here right?

Reply:
I don't think I understand. I am testing for `SLE_12_SP4` or am I missing something?

Reply:
My bad. I think I read the code wrong. Changes looks good.

Reply:
No worries :smile: Thanks!